client/src/components/CardContainer/CardContainer.module.scss

```diff
@@ -0,0 +1,100 @@
+.cardContainer {
+  max-width: 1150px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.headerContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .header {
+    font-size: 24px;
+    color: #0499DD;
+    margin-bottom: 0;
+  }
+
+  .allTours {
+    font-size: 12px;
+    color: #282828;
+  }
+}
+
+.swiperContainer {
+  position: relative;
+  margin: 70px auto 0 auto;
+
+  .btnPrev {
+    background-color: white;
+    height: 60px;
+    width: 60px;
+    border-radius: 50px;
+    position: absolute;
+    cursor: pointer;
+    z-index: 2;
+    top: 50%;
+    margin-top: -30px;
+    margin-left: -30px;
+
+    &::after {
+      content: url("../../../src/svg/hotDeals/btn-prev.svg");
+      position: absolute;
+      width: 10px;
+      top: 50%;
+      left: 50%;
+      margin-left: -2px;
+      margin-top: -9px;
+      transform: scaleX(-1);
+    }
+  }
+
+  .btnNext {
+    background-color: white;
+    height: 60px;
+    width: 60px;
+    border-radius: 50px;
+    position: absolute;
+    cursor: pointer;
+    z-index: 2;
+    top: 50%;
+    right: -30px;
+    margin-top: -30px;
+    margin-left: 100%;
+
+    &::after {
+      content: url("../../../src/svg/hotDeals/btn-prev.svg");
+      position: absolute;
+      width: 10px;
+      top: 50%;
+      left: 50%;
+      margin-left: -6px;
+      margin-top: -9px;
+    }
+  }
+}
+
+@media only screen and (min-width: 300px) {
+  .swiperContainer {
+    width: 265px;
+  }
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
Media queries can be mentioned in a styled component as follows:

`const MyComponent = styled("div")({
"[@media only screen and (min-width: 300px)]": {
  .swiperContainer {
    width: 265px;
  }
}
}
[
})`